### ISSUE TITLE
Ignore/detect symlink loops in folders plugin

### ISSUE THREAD

**Issue by Ellpeck** [nicolargo/glances#2494]
**Is your feature request related to a problem? Please describe.**
Currently, the folders plugin displays a `?` for any folders that have symlink loops in them, like the `/var/lib/docker` directory, for example. 

When enabling the debug flag, the following info is displayed in the log file:
```
-- DEBUG -- Cannot get folder size (/var/lib/docker). Error: [Errno 40] Symbolic link loop: [...]
```

**Describe the solution you'd like**
I think it would make sense if the error described above would be caught in the folders plugin, and the true size of the folder without the symlink loop would be displayed, instead of a `?`. 

Tools like `ncdu` already do this, and Glances' file system plugin seems to deal fine with symlinks already.

Thanks so much for this great tool, and thanks for possibly considering my suggestion!! ❤️

**Comment by nicolargo** [comment]
Hi @Ellpeck 

Thanks for the issue report :)

I just push a patch on the develop branch in order to avoid Glances Folders plugin to follow symbolic link. Glances has now the same behavor than ncdu.

Nicolas

**Comment by Ellpeck** [comment]
Hi! I tried the change and it doesn't appear to error based on symlinks anymore now, thanks so much!!

However, I found another issue, and I hope you don't mind me reusing this issue report for it since it's in the same vein: The folders plugin errors when it fails to detect the size of the `proc` directory on Linux, which causes an issue for displaying the size of the docker volume directory, since those also contain versions of `proc`). 

Obviously, this makes sense and is in line with other tools, but those tools just display an error and continue scanning directory sizes instead of failing altogether. This is `ncdu` for example:
```
Warning: error scanning /proc/579310/task/579311/fd/23
    some directory sizes may not be correct
```

Potentially, it would also make sense for the `folders` plugin to either display an error and continue scanning rather than failing outright, or to be able to specify a set of file names (maybe through a regular expression or a glob pattern?) that should be ignored when scanning.
